--- origsrc/libgpod-0.6.0/Makefile.am	2007-11-10 04:19:31.000000000 -0600
+++ src/libgpod-0.6.0/Makefile.am	2007-12-22 18:46:15.843750000 -0600
@@ -3,6 +3,8 @@
 # build documentation when doing a distcheck.
 DISTCHECK_CONFIGURE_FLAGS = --enable-gtk-doc
 
+ACLOCAL_AMFLAGS = -I m4
+
 pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = libgpod-1.0.pc
 
--- origsrc/libgpod-0.6.0/bindings/python/Makefile.am	2007-07-23 14:51:24.000000000 -0500
+++ src/libgpod-0.6.0/bindings/python/Makefile.am	2007-12-22 18:46:15.843750000 -0600
@@ -41,8 +41,9 @@
 gpod_LTLIBRARIES = _gpod.la
 nodist__gpod_la_SOURCES = gpod_wrap.c
 _gpod_la_CPPFLAGS = $(PYTHON_INCLUDES) $(LIBGPOD_CFLAGS) $(INCLUDES)
-_gpod_la_LDFLAGS = -module -avoid-version
-_gpod_la_LIBADD = $(LIBGPOD_LIBS) $(top_builddir)/src/libgpod.la
+_gpod_la_LDFLAGS = -module -avoid-version -no-undefined
+_gpod_la_LIBADD = $(LIBGPOD_LIBS) $(top_builddir)/src/libgpod.la \
+	-L$(libdir)/python$(PYTHON_VERSION)/config -lpython$(PYTHON_VERSION)
 
 gpod_doc.i: $(srcdir)/gpod_doc.i.in $(srcdir)/gtkdoc-to-swig.xsl
 	cat $< > $@
